test/test_resize.cpp

```diff
@@ -1124,6 +1124,109 @@ TEST_F(ResizeTest, FusionResizeSlice5) {
   testValidate(&fusion, cg_outputs, aten_inputs, {t2, t4}, __LINE__, __FILE__);
 }
 
+// Test slice with a variety of constant ranges
+TEST_F(NVFuserTest, FusionResizeSliceConstantShmoo_CUDA) {
+  for (auto [start, stop] : std::vector<std::pair<int64_t, int64_t>>(
```

Comment:
Should we use the same set of slices as `FusionResizeSliceInputShmoo_CUDA`?

Reply:
Yes that's good now. The reason I didn't originally is just that it slows down the test a lot since we need to recompile for each slice.